ipaclient/install/client.py

```diff
@@ -2557,13 +2497,22 @@ def _install(options):
         tasks.set_hostname(options.hostname)
 
     if options.conf_ntp:
-        # Attempt to configure and sync time with NTP server (chrony).
-        sync_time(options, fstore, statestore)
+        # Attempt to configure and sync time with NTP server.
+        if not createntp.sync_time_client(
+                fstore, statestore, cli_domain,
+                options.ntp_servers, options.ntp_pool):
+            print("Warning: IPA client was unable to sync time "
+                  "with IPA server!")
+            print("         Time synchronization is required for IPA "
+                  "to work correctly!")
+        else:
+            print("Time successfully synchronized with IPA server")
     elif options.on_master:
         # If we're on master skipping the time sync here because it was done
         # in ipa-server-install
         logger.debug("Skipping attempt to configure and synchronize time with"
-                     " chrony server as it has been already done on master.")
+                     " %s server as it has been already done on master.",
+                     detect_time_server)
```

Comment:
Probably missing `()` but have in mind previous https://github.com/freeipa/freeipa/pull/2169#discussion_r232630498
```suggestion
                     detect_time_server())
```